controllers/podhttpchaos/controller.go

```diff
@@ -58,6 +58,11 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		return ctrl.Result{}, nil
 	}
 
+	if obj.ObjectMeta.Generation <= obj.Status.ObservedGeneration && obj.Status.FailedMessage == "" {
```

Comment:
Maybe we could write `PodXXXChaosDelegate` (with a better name), as the controllers of `podiochaos`, `podhttpchaos`, and `podnetworkchaos` are nearly the same in the future.

This PR is fine to merge.

Reply:
Looks good